examples/protocol_greetings_with_instant_server.rs

```diff
@@ -33,6 +36,7 @@ impl ProtocolEntry for Receiver {
 
 #[tokio::main]
 async fn main() -> Result<(), Box<dyn std::error::Error + Send + Sync + 'static>> {
+    let lr = LocalRegistry::new().await;
```

Comment:
can we point out in this example, that, the instant server will automatically use the local registry when there is one？